test/spec-testing.js

```diff
@@ -0,0 +1,70 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/loopback-next-hello-world
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+'use strict';
+
+const expect = require('@loopback/testlab').expect;
+const HelloWorldApp = require('../lib/application').HelloWorldApp;
+const http = require('http');
+const Dredd = require('dredd');
+const port = 3030; // use this port for Dredd testing only
+const localhost = 'http://localhost:' + port;
+const dreddConfig = {
+  server: localhost + '/', // base path to the end points
```

Comment:
If this is a global config between, maybe we can put it in a file and load it from there?